Engine/src/Window/Window.bf

```diff
@@ -62,10 +62,10 @@ namespace SteelEngine.Window
 				Log.Fatal("Could not initialize GLFW");
 			}
 
-			Glfw.WindowHint(GlfwWindow.Hint.ContextVersionMajor, 3);
-			Glfw.WindowHint(GlfwWindow.Hint.ContextVersionMinor, 3);
+			/*Glfw.WindowHint(GlfwWindow.Hint.ContextVersionMajor, 4);
+			Glfw.WindowHint(GlfwWindow.Hint.ContextVersionMinor, 6);
 			Glfw.WindowHint(GlfwWindow.Hint.OpenGlProfile, .CoreProfile);
-			Glfw.WindowHint(GlfwWindow.Hint.OpenGlForwardCompat, Glfw.TRUE);
+			Glfw.WindowHint(GlfwWindow.Hint.OpenGlForwardCompat, Glfw.TRUE);*/
```

Comment:
This should have a comment explaining why we aren't using these hints for now.

Reply:
@RogueMacro Any update on this?

Reply:
It wasn’t fixed at the moment of reviewing this, but it might be with the new library. I will look into it later today. Thanks for pointing this out